test/commands/project/project.js

```diff
@@ -346,32 +346,49 @@ describe('project', function() {
       project.create(name, { framework: 'connect' }, done);
     });
 
-
-    it('should pass test-module options', function(done) {
-      var options = { testModule: 'request' };
+    it('should err when given invalid test-module options', function(done) {
+      var options = { testModule: 'wrong'};
       project.generateTest(projPath, options, function(err) {
-        should.not.exist(err);
-        fs.existsSync(path.resolve(projPath, 'test/helloStub.js')).should.be.ok;
+        should.exist(err);
         done();
       });
     });
 
-    it('should err when given invalid test-module options', function(done) {
-      var options = { testModule: 'wrong'};
+    it('should pass test-module options', function(done) {
+      var options = { testModule: 'request'  };
+      var c = 0;
```

Comment:
Is this a unused variable? please remove it and make sure ESLint catches stuff like this.